build/proxy.conf.localdev.js

```diff
@@ -12,6 +12,16 @@ const PROXY_CONFIG = {
     "ws": true,
     "secure": false,
     "changeOrigin": true,
+  },
+  "/api": {
+    "target": {
```

Comment:
Not all of the `/pp` endpoints have been migrated to `/api`, just the ones outlined in #4532. So replacing `/pp` with `/api` will break the leftover endpoints.

Reply:
@nwmac hmm, should I add some fancy logic to `./build/dev-setup.js` to add an `/api` section to `proxy.conf.localdev.js` if it's missing, or should this be left to the developer?

Reply:
@ikapelyukhin I think we should add the fancy logic OR if that's a i real pain, a simpler option is:

- if the file `proxy.conf.js` does not exist, create it as we do now
- if `proxy.conf.js` exists but has not been changed (same MD5), then overwrite it with the new version
- otherwise, warn the user that they need to update it manyally

Reply:
OK, sounds reasonable, done.